README.md

```diff
@@ -1 +1,3 @@
-# Endjin.RecommendedPractices.AzureDevopsPipelines.Python
\ No newline at end of file
+# Endjin.RecommendedPractices.AzureDevopsPipelines.Python
+
+Repository to capture best practice tooling for Python based developments.
```

Comment:
"developments" (plural) has a slightly different connotation from "development" - is that intentional?

Reply:
Good point Ian, I guess it reflects that I'm still not sure how this repo will evolve : still getting my head around how endjin evolves IP in this way.

The motivation is to support a range of of different use cases where Python could be leveraged within the wider modern data platform vision.

The initial objective is to provide a blueprint for creation of a production grade Python package that can be imported into multiple use cases.

Then this could be extended to provide blueprints for common use cases, examples being:
- Python notebooks in Synapse;
- Deploying Python within an Azure function;
- Use of Python for visualisation in PowerBI;
- Python based Spark job on Synapse.

Not sure if this would sit in one repo or many.